lib/views/LineGraph.js

```diff
@@ -25,33 +26,47 @@ class LineGraph extends Graph {
             .y(d => this.yRange(d.y_value))
             .interpolate(this.interpolate);
 
-        var line = this.d3Svg.selectAll('path.line')
-                    .data([this.dataSeries.data]);
+        var data = Utils.splitOnMissingValues(this.dataSeries.data, ['x_value', 'y_value']);
```

Comment:
Yeah, I think there's far too much work that goes into creating simple charts. It'd be sweet to minimize the amount of moving parts.